docs/docs/explanations/architecture.md

```diff
@@ -8,61 +8,111 @@ sidebar_position: 2
+
+Example: A Kurtosis engine container running in Docker:
+
+```console
+bfb5627ff511   kurtosistech/engine:0.70.7                        "/bin/sh -c ./kurtos…"   10 hours ago   Up 10 hours   0.0.0.0:9710-9711->9710-9711/tcp                   kurtosis-engine--f84ce1f4c5ea410080e774cfea0ea0a4
+```
+
 Services
 --------
-Enclaves contain distributed applications, and distributed applications have services. A service in Kurtosis is a container that exposes ports, and services may depend on other services (e.g. an API server depending on a database). Each enclave can have an arbitrary numbers of services, limited only by the underlying hardware.
+Enclaves contain distributed applications, and distributed applications are composed of services. In Kurtosis, a service is a container that exposes ports. Services may also depend on other services (e.g. an API server depending on a database). Each enclave can have an arbitrary numbers of services, limited only by the underlying hardware.
 
 Example: A pair of Nginx services running inside an enclave called `test`, as reported by the Kurtosis CLI:
 
 ```
-Enclave UUID:                         2e42f9fd7b854eabb04f71a15bd1b55f
-Enclave Name:                         test
-Enclave Status:                       RUNNING
+UUID:                         2e42f9fd7b854eabb04f71a15bd1b55f
+Name:                         test
+Status:                       RUNNING
 Creation Time:                        Thu, 24 Nov 2022 11:11:34 -03
-API Container Status:                 RUNNING
-API Container Host GRPC Port:         127.0.0.1:60768
-API Container Host GRPC Proxy Port:   127.0.0.1:60769
 
 ========================================== User Services ==========================================
 GUID                ID       Ports                             Status
 nginx-1669299161    nginx    http: 80/tcp -> 127.0.0.1:60785   RUNNING
 nginx2-1669299176   nginx2   http: 80/tcp -> 127.0.0.1:60794   RUNNING
 ```
 
+API Container
+-------------
+The second type of container that Kurtosis creates is the API container. One API container is created per enclave, and each API container is responsible for managing interactions with its own enclave. All manipulation of an enclave's contents happens via API calls to the enclave's API container. 
+
+Example: a Kurtosis API container running in Docker:
+
+```
+3c0b6ab7bb85   kurtosistech/core:0.70.7                          "/bin/sh -c ./api-co…"   20 hours ago    Up 20 hours    0.0.0.0:58419->7443/tcp, 0.0.0.0:58418->7444/tcp   kurtosis-api--6babc3090ad04184b2094901a7ead7b4
+```
+
+Starlark
+--------
+Distributed system definitions are complex. Therefore, there are many, many ways to instantiate, configure, and manipulate an enclave. To provide the required power, manipulations to an enclave are expressed in the form of [Starlark programs][starlark-reference]. 
```

Comment:
One nit: this is the first time I've read about `Starlark programs`. Now the user has to reason about "Starlark Instructions", "Starlark Types", and now "Starlark Programs". Can we consolidate?

```suggestion
Distributed system definitions are complex. Therefore, there are many, many ways to instantiate, configure, and manipulate an enclave. To provide the required power, manipulations to an enclave are expressed using [Starlark][starlark-reference] scripts.
```